topics.json

```
@@ -98,6 +98,7 @@
   "snmp-authentication": "/configuration/devices/snmp-trap#authentication-protocols",
   "snmp-privacy": "/configuration/devices/snmp-trap#privacy-protocols",
 
+  "appendix-windows-event-ids": "/appendix/windows-event-ids",
```

Biome (2.1.2)

[error] The key appendix-windows-event-ids was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)